test/game.test.js

```diff
@@ -467,5 +468,16 @@ describe("Game", () => {
                 expect(accessibilify).toHaveBeenCalledWith({ config: { id: 4, ariaLabel: "Continue" } });
             });
         });
+        describe("shop button", () => {
+            shop.launchShopOverlay = jest.fn();
+
+            beforeEach(() => game.create());
```

Comment:
Don't know if I would put anything in a beforeEach until there is more than one test unless I was expecting more?

Reply:
Good point! Will tweak.